features/importing/archivist.feature

```diff
@@ -81,8 +84,11 @@ Feature: Archivist bulk imports
       | user1 | a@ao3.org |
     When I go to the import page
       And I import the work "http://ao3testing.dreamwidth.org/593.html" by "name1" with email "a@ao3.org" and by "name2" with email "b@ao3.org"
-    Then 1 email should be delivered to "a@ao3.org"
+    When the system processes jobs
```

Comment:
Is this step necessary? We're planning to remove it: https://otwarchive.atlassian.net/browse/AO3-5722